src/ripple/app/paths/cursor/Liquidity.cpp

```diff
@@ -45,7 +45,7 @@ TER PathCursor::liquidity () const
             << "reverseLiquidity< "
             << "nodeIndex=" << pc.nodeIndex_
             << " resultCode=" << transToken (resultCode)
-            << " transferRate_=" << pc.node().transferRate_
+            << " transferRate_=" << *pc.node().transferRate_
```

Comment:
I _think_ optional already asserts in debug mode. I was more worried about actual crashes in production due to logging.